packages/@lwc/engine-core/src/framework/secure-template.ts

```diff
@@ -34,6 +34,8 @@ export function registerTemplate(tpl: Template): Template {
     // on top of stylesheetToken for anyone who is accessing the old internal API.
     // Details: https://salesforce.quip.com/v1rmAFu2cKAr
     defineProperty(tpl, 'stylesheetTokens', {
+        enumerable: true,
+        configurable: true,
```

Comment:
The `configurable` allows us to define another descriptor on top of this descriptor, which helps isolate the freezing/warning logic to `freezeTemplate`.

The `enumerable` is just for consistency's sake, since the same thing happens when you do `tmpl.stylesheets = []`.